packages/noya-state/src/reducers/themeReducer.ts

```diff
@@ -43,6 +43,7 @@ export type ThemeAction =
       mode?: SetNumberMode,
     ]
   | [type: `addSwatch`, name?: string, color?: Sketch.Color, id?: string]
+  | [type: `addGradientStyle`, name: string, value: Sketch.Gradient]
```

Comment:
Maybe `addGradientAsset` or `addGradient`? A "style" should usually refer to a `Sketch.Style`